eth/tracers/native/prestate.go

```diff
@@ -42,15 +42,18 @@ func init() {
 type stateMap = map[common.Address]*account
 
 type account struct {
-	Balance *big.Int                    `json:"balance,omitempty"`
-	Code    []byte                      `json:"code,omitempty"`
-	Nonce   uint64                      `json:"nonce,omitempty"`
-	Storage map[common.Hash]common.Hash `json:"storage,omitempty"`
-	empty   bool
+	Balance         *big.Int                    `json:"balance,omitempty"`
+	Code            []byte                      `json:"code,omitempty"`
+	Nonce           uint64                      `json:"nonce,omitempty"`
+	Storage         map[common.Hash]common.Hash `json:"storage,omitempty"`
```

Comment:
How about adding an empty line after this one?
All lines above exist in the original geth, so adding an empty line here would avoid changing those lines, which can simplify the process of updating this fork with changes from the original geth in the future.

Reply:
I agree